workspace.json

```diff
@@ -5014,8 +5041,7 @@
             "main": "apps/icelandic-names-registry/backend/src/main.ts",
             "tsConfig": "apps/icelandic-names-registry/backend/tsconfig.app.json",
             "webpackConfig": "apps/icelandic-names-registry/backend/webpack.config.js",
-            "assets": [
-              {
+            "assets": [{
```

Comment:
Where do all these changes come from. Is this formatted by prettier?

Reply:
My local prettier seems to be configured incorrectly for .json files